bom/application/pom.xml

```diff
@@ -34,7 +34,6 @@
         <microprofile-opentracing-api.version>1.3.3</microprofile-opentracing-api.version>
         <microprofile-reactive-streams-operators.version>1.0.1</microprofile-reactive-streams-operators.version>
         <microprofile-rest-client.version>1.4.1</microprofile-rest-client.version>
-        <smallrye-common.version>1.4.0</smallrye-common.version>
```

Comment:
Why this was removed and not updated to 1.4.1?

Reply:
MY bad, something went off in the last rebase.

Reply:
Fixed